src/main/java/be/seeseemelk/mockbukkit/entity/MushroomCowMock.java

```diff
@@ -79,4 +91,47 @@ public void setStewEffect(@Nullable PotionEffectType type)
 		return EntityType.MUSHROOM_COW;
 	}
 
+	@Override
+	public void shear(Sound.@NotNull Source source)
+	{
+		WorldMock world = (WorldMock) this.getWorld();
```

Comment:
Does this not already return a WorldMock?
If not, can we change it so that the cast is unnecessary here?